bool garateia
bool val_helio
bool comutador
